app/cafe.py

```diff
@@ -0,0 +1,20 @@
+# The Cafe class should be defined in cafe.py module.
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You are not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Expired vaccination date")
+        if not visitor["wearing_a_mask"] or visitor["wearing_a_mask"] is False:
```

Comment:
This line checks if `visitor['wearing_a_mask']` is `False` or `None`. You can simplify this by just checking `if not visitor['wearing_a_mask']`. This will return `True` if `visitor['wearing_a_mask']` is `False` or `None`.